Build failure: glaxnimate

### Steps To Reproduce

Steps to reproduce the behavior:
I wanted to update the system and got the error.
then tried with `nix shell nixpkgs#glaxnimate` and got the same error

### Build log

```
error: builder for '/nix/store/03dwgflycwl7p03wdhcfy8ji66kxp006-glaxnimate-0.5.1.drv' failed with exit code 2;
       last 10 log lines:
       >       |     ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > In file included from /nix/store/ffll6glz3gwx342z0ch8wx30p5cnqz1z-python3-3.11.5/include/python3.11/Python.h:96:
       > /nix/store/ffll6glz3gwx342z0ch8wx30p5cnqz1z-python3-3.11.5/include/python3.11/sysmodule.h:14:38: note: declared here
       >    14 | Py_DEPRECATED(3.11) PyAPI_FUNC(void) PySys_SetArgvEx(int, wchar_t **, int);
       >       |                                      ^~~~~~~~~~~~~~~
       > make[2]: *** [external/QtAppSetup/src/CMakeFiles/QtAppSetup.dir/build.make:300: external/QtAppSetup/src/CMakeFiles/QtAppSetup.dir/app/scripting/python/python_engine.cpp.o] Error 1
       > make[2]: *** Waiting for unfinished jobs....
       > make[2]: *** [external/QtAppSetup/src/CMakeFiles/QtAppSetup.dir/build.make:314: external/QtAppSetup/src/CMakeFiles/QtAppSetup.dir/app/scripting/python/register_machinery.cpp.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:528: external/QtAppSetup/src/CMakeFiles/QtAppSetup.dir/all] Error 2
       > make: *** [Makefile:156: all] Error 2
       For full logs, run 'nix log /nix/store/03dwgflycwl7p03wdhcfy8ji66kxp006-glaxnimate-0.5.1.drv'.

```

### Additional context


### Notify maintainers
@tobiasBora 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.58, NixOS, 23.11 (Tapir), 23.11.20231024.8efd5d1`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.1`
 - channels(tiggax): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/applications/video/glaxnimate/default.nix
+++ b/pkgs/applications/video/glaxnimate/default.nix
@@ -37,19 +37,20 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "glaxnimate";
-  version = "0.5.1";
+  version = "0.5.4";
 
   src = fetchFromGitLab {
     owner = "mattbas";
     repo = "${pname}";
     rev = version;
-    sha256 = "G4ykcOvXXnVIQZUYpRIrALtDSsGqxMvDtcmobjjtlKw=";
+    sha256 = "sha256-8oHJCQdP2xxSSDM0MDkSrG89WgCtMKm1AKlddnq3gig=";
     fetchSubmodules = true;
   };
 
   nativeBuildInputs = [
     cmake
     wrapQtAppsHook
+    qttools
   ];
 
   buildInputs = [
@@ -65,6 +66,12 @@ stdenv.mkDerivation rec {
     python3WithLibs
   ];
 
+  # Translation needs to be separately compiled
+  # https://gitlab.com/mattbas/glaxnimate/-/issues/648
+  postBuild = ''
+    make translations
+  '';
+  
   qtWrapperArgs = [ ''--prefix PATH : ${python3WithLibs}/bin'' ];
 
   passthru.tests.version = lib.optionalAttrs stdenv.isLinux (testers.testVersion {

--- a/pkgs/applications/video/glaxnimate/default.nix
+++ b/pkgs/applications/video/glaxnimate/default.nix
@@ -68,9 +68,7 @@ stdenv.mkDerivation rec {
 
   # Translation needs to be separately compiled
   # https://gitlab.com/mattbas/glaxnimate/-/issues/648
-  postBuild = ''
-    make translations
-  '';
+  buildFlags = ["translations"];
   
   qtWrapperArgs = [ ''--prefix PATH : ${python3WithLibs}/bin'' ];
 

--- a/pkgs/applications/video/glaxnimate/default.nix
+++ b/pkgs/applications/video/glaxnimate/default.nix
@@ -68,8 +68,8 @@ stdenv.mkDerivation rec {
 
   # Translation needs to be separately compiled
   # https://gitlab.com/mattbas/glaxnimate/-/issues/648
-  buildFlags = ["translations"];
-  
+  buildFlags = [ "translations" ];
+
   qtWrapperArgs = [ ''--prefix PATH : ${python3WithLibs}/bin'' ];
 
   passthru.tests.version = lib.optionalAttrs stdenv.isLinux (testers.testVersion {

